webpack.config.js

```diff
@@ -0,0 +1,161 @@
+fs.readdirSync(path.resolve(__dirname, './src/TWIG/Pages')).forEach((file) => {
+  const page = new HtmlWebpackPlugin({
+    filename: file.replace('.twig', '.html'),
+    template: path.resolve(__dirname, `./src/TWIG/Pages/${file}`),
+    hash: true,
+    // minify: {
+    //   collapseWhitespace: isProd,
+    // },
+  });
+  HWPConfig.push(page);
+});
+
+module.exports = {
+  context: path.resolve(__dirname, 'src'),
+  entry: { main: ['@babel/polyfill', './index.js'] },
+  output: {
+    filename: filename('js'),
+    path: path.resolve(__dirname, 'dist'),
+  },
+  optimization: optimization(),
+  devServer: {
+    port: 3000,
+    hot: isDev,
+  },
+  devtool: isDev ? 'source-map' : false,
+  plugins: [
+    // new HtmlWebpackPlugin({
+    //   template: './index.html',
+    //   minify: {
+    //     collapseWhitespace: isProd,
+    //   },
+    // }),
+    ...HWPConfig,
+    new CleanWebpackPlugin(),
+    new MiniCssExtractPlugin({ filename: filename('css') }),
+  ],
+  module: {
+    rules: [
+      {
+        test: /\.(sa|sc|c)ss$/,
+        exclude: /node_modules/,
+        use: [
+          isDev ? 'style-loader' : MiniCssExtractPlugin.loader,
+          'css-loader',
+          {
+            loader: 'postcss-loader',
+            options: {
+              postcssOptions: {
+                plugins: ['postcss-preset-env'],
+              },
+            },
+          },
+          {
+            loader: 'sass-loader',
+            options: {
+              sourceMap: true,
+            },
+          },
+        ],
+      },
+      {
+        test: /\.js$/,
+        exclude: /node_modules/,
+        use: {
+          loader: 'babel-loader',
+          options: {
+            presets: ['@babel/preset-env'],
+            plugins: [
+              '@babel/plugin-proposal-class-properties',
+              '@babel/plugin-proposal-object-rest-spread',
+            ],
+          },
+        },
+      },
+      {
+        test: /\.ts$/,
+        exclude: /node_modules/,
+        use: {
+          loader: 'babel-loader',
+          options: babelOptions('@babel/preset-typescript'),
+        },
+      },
+      {
+        test: /\.jsx$/,
+        exclude: /node_modules/,
+        use: {
+          loader: 'babel-loader',
+          options: babelOptions('@babel/preset-react'),
+        },
+      },
+      {
+        test: /\.twig$/,
+        exclude: /node_modules/,
+        use: [
+          'raw-loader',
+          {
+            loader: 'twig-html-loader',
+            options: { cache: true },
+          },
+        ],
+      },
+      {
+        test: /\.(png|svg|jpg|gif)$/,
+        loader: 'file-loader',
+        options: {
+          name: isDev ? '[path][name].[ext]' : '[contenthash].[ext]',
+        },
+      },
+      {
+        test: /\.(woff|woff2|eot|ttf|otf)$/,
+        loader: 'file-loader',
+        options: {
+          name: isDev ? '[path][name].[ext]' : '[contenthash].[ext]',
+        },
+      },
+    ],
+  },
+};
```

Comment:
Please make sure you have at least an SCSS folder for SCSS files for your styles